projects/js-packages/charts/package.json

```diff
@@ -74,6 +74,7 @@
 			"import": "./dist/providers/index.js",
 			"require": "./dist/providers/index.cjs"
 		},
+		"./styles.css": "./dist/index.css",
```

Comment:
Feels we'll need to add styles exports for each entry as well, which would be quite a few 🤔 Will only expose this one for now.